app/components/organization/SettingsMenu.js

```diff
@@ -101,6 +101,18 @@ class SettingsMenu extends React.Component {
         )
       },
       {
+        allowed: "teamAdmin",
```

Comment:
Hrm... I don't think `teamAdmin` is appropriate for this one (since that'll allow non-org admins to view logs). We'll need to add a new permission, but we can do that in another PR.

So for now, let's use `agentTokenView`